lib/protocol/routes.js

```diff
@@ -597,6 +597,22 @@ const METHOD_MAP = {
   '/wd/hub/session/:sessionId/element/:elementId/property/:name': {
     GET: {command: 'getProperty'}
   },
+  '/wd/hub/session/:sessionId/appium/device/set_clipboard': {
+    POST: {command: 'setClipboard', payloadParams: {
+      required: ['content'],
+      optional: [
+        'contentType',
+        'label',
+      ]},
+    }
+  },
+  '/wd/hub/session/:sessionId/appium/device/get_clipboard': {
+    POST: {command: 'getClipboard', payloadParams: {
```

Comment:
This being a `POST` request seems weird. Is the `contentType` necessary?

Reply:
this is optional, but I'd like to keep it, so one can filter clipboard content. Also, this could be extended later to get the content of multiple clipboard items and not only the primary one

Reply:
Alright.